pkg/storage/storage.go

```diff
@@ -119,6 +127,12 @@ type ReadStartingWithUserFilter struct {
 	UserFilter []*openfgapb.ObjectRelation
 }
 
+type ReadUsersetTuplesFilter struct {
```

Comment:
I noticed where we use this we have to split it everywhere. Can we please break it apart so it's more explicit?

```
type ReadUsersetTuplesFilter struct {
    ObjectType string
    ObjectID string
    ...
}
```

Reply:
> where we use this we have to split it everywhere

only in `mysql.go` and `postgres.go`. Every other place (Check, the tests) don't have to. I'm gonna keep this as is

Reply:
Is ObjectID an id or an object though? If it is an object I would prefer to name it as such.

Reply:
currently, `ObjectID` is e.g. `doc:budget`. Do you prefer just `Object`?

Reply:
If we have a `TupleKey` and we call `GetObject` on it, then it will return a `doc:budget` thing, so, in my case anyway, I would prefer `Object`.